editor/components/url-input/index.js

```diff
@@ -239,4 +240,21 @@ class UrlInput extends Component {
 	}
 }
 
-export default withSpokenMessages( withInstanceId( UrlInput ) );
+export class UrlInput extends Component {
```

Comment:
Should we wrap this one in `withSpokenMessages( withInstanceId(` ?

Reply:
>Should we wrap this one in `withSpokenMessages( withInstanceId(` ?

The canonical `URLInput` has these higher-order components still, and this one is just a pass-through that has no functionality of its own, so it shouldn't be necessary.

Reply:
From my understanding you're not using the `URLInput` that's wrapped with those, unless I'm misreading :)

Reply:
Ah, I see now. You're right.

...The dangers of applying higher-order components as part of the `export default` statement 😬

Reply:
Should be fixed now. Verified `instanceId` on the passthrough component is assigned into its ID.